joshpham97/soen487-restful-repository
# Setup Frontend

Created a `webapp` module setting up React
<br />

<ins>What you need:</ins>
- Nodejs
- npm

If you don't have any then you could just install this: https://nodejs.org/en/ (includes both nodejs and npm)
<br />

<ins>How to start app:</ins>
1. run the album service server
2. open cmd or use built-in terminal in IDE
3. cd to the `webapp` directory
4. run command `npm install`
5. then run `npm start`
<br />

<ins>What to test:</ins>
- Click the links in the navbar
- Make sure each page displays correctly
- Make sure the example and example2 pages displays the list of albums correctly
<br />

<ins>Screenshots to compare:</ins>
![Capture](https://user-images.githubusercontent.com/16867166/108612849-324dc180-73ba-11eb-951a-e1021bfc4d07.JPG)
![Capture](https://user-images.githubusercontent.com/16867166/108612946-ec452d80-73ba-11eb-9c10-ba777c415a3c.JPG)
![Capture](https://user-images.githubusercontent.com/16867166/108612965-08e16580-73bb-11eb-8edf-54a131ba07fe.JPG)
